applications/sasquatch/values-usdfdev.yaml

```diff
@@ -123,6 +123,12 @@ kafka-connect-manager:
         connectInfluxDb: "lsst.lf"
         topicsRegex: "lsst.lf.*"
         tags: benchmark_env,module,benchmark_type
+      lsstprompt:
+        enabled: true
+        timestamp: "timestamp"
```

Comment:
Do we need to do anything special to ensure there is a timestamp column, or do `analysis_tools` and `SasquatchDatastore` already enforce that?

Reply:
`analysis_tools` already add the timestamp column and a timestamp value with the right format.